src/build/build_implicitconstraints.jl

```diff
@@ -82,12 +149,12 @@ function create_utility_expr(jm, dm::DemandFunction)
                         :(
                             $( Θ(dm, demand) ) *
                             # TODO #71 Figure out why the commented version doesn't work, it matches paper
-                            # (
-                            #     $(jm[get_final_demand_name(demand)]) / $(demand.quantity)
-                            # )^(
-                            #     ($(dm.elasticity)-1)/$(dm.elasticity)
-                            # )
-                            ($(demand.quantity)/$(demand.quantity))^(($(dm.elasticity)-1)/$(dm.elasticity))
+                            (
+                            $(swap_our_param_with_val(demand.commodity)) # Attempted fix 2022-12-21
+                                # $(demand.quantity)  # 71 version
+                            #  / $(demand.quantity))  # 71 version
+                          /$(get_commodity_benchmark(demand.commodity))) # Attempted fix 2022-12-21
+                            ^(($(dm.elasticity)-1)/$(dm.elasticity))
```

Comment:
I think this entire diff doesn't actually change anything? `swap_our_param_with_val(demand.commodity)` is the same as `get_commodity_benchmark(demand.commodity)`, so we again just have a fraction here that cancels out to 1?